src/cluster.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import * as MapboxGL from 'mapbox-gl';
```

Comment:
if by using the ambient definition you mean this
```
/// <renference type="..."/>
```
I prefer not to use it because if i understand it well it expose globaly all the types of your reference in your project using this right ? even though I think it is difficult to track where the type come from